build.gradle

```diff
@@ -2,8 +2,11 @@
 plugins {
     id 'java-library'
     id "org.openmicroscopy.project" version "5.5.0-m5"
+    id "application"
 }
 
+mainClassName = "omero.gateway.Gateway"
```

Comment:
I'm guessing you want to set the main-class in the jar manifest? 

```
jar {
    manifest {
        attributes("Main-Class": "omero.gateway.Gateway")
    }
}
```

Reply:
The class doesn't have a `public static main` though so ...

Reply:
In that case just document in the readme, with an example preferably, that Gateway is what a dev wants to focus on as their "starting point".  Setting `mainClassName` doesn't have any affect on the jar manifest.

Reply:
currently there is no readme and changelog. This will be added in follow-up PR